@xen-orchestra/fs/src/abstract.js

```diff
@@ -612,6 +607,19 @@ export default class RemoteHandlerAbstract {
   ): Promise<void> {
     throw new Error('Not implemented')
   }
+
+  async __closeFile(fd: FileDescriptor): Promise<void> {
+    await timeout.call(this._closeFile(fd.fd), this._timeout)
+  }
+
+  async __openFile(path: string, flags: string): Promise<FileDescriptor> {
+    path = normalizePath(path)
+
+    return {
+      fd: await timeout.call(this._openFile(path, flags), this._timeout),
+      path,
+    }
+  }
```

Comment:
IMHO, these methods are a layer between public and implementation methods, the location of the section should reflect this.